src/tablo/Header/header.scss

```diff
@@ -5,26 +5,31 @@
 
 .tablo {
   .public_fixedDataTable_header {
-    border-width: $border-width;
-    border-color: $border-color;
     background: $header-background;
+    border-width: 0; // to reset
+    border-bottom: $header-row-border-width solid $header-row-border-color;
 
-    .public_fixedDataTableCell_cellContent {
-      background: $header-background;
-      color: $header-color;
-    }
-  }
+    .public_fixedDataTableCell_main {
+      .tablo-header {
+        color: $header-color;
+        padding: 0 5px;
 
-  .tablo-header {
-    background: $header-background;
-    color: $header-color;
-    padding: 0 5px;
+        /*
+          > div, > div > div {
```

Comment:
Also this is outdated. I removed it in the end. It was commented out since long time and we haven't had issues about vertical centering of the header's content on IE